src/plugins/intel_cpu/tests/functional/utils/cpu_test_utils.cpp

```diff
@@ -119,6 +119,12 @@ ov::PrimitivesPriority CPUTestsBase::impls2primProiority(const std::vector<std::
     return ov::PrimitivesPriority(str);
 }
 
+inline std::string getExecValue(const ov::Node::RTMap& rtInfo, const std::string& paramName) {
```

Comment:
```suggestion
inline std::string getRuntimeValue(const ov::Node::RTMap& rtInfo, const std::string& paramName) {
```

Reply:
Renamed. I also noticed that this logic is used in several places, so I moved a function declaration to hpp and reused this function in that places